Sketcher/OffsetInSketcher.FCMacro

```diff
@@ -0,0 +1,431 @@
+__name__   = "OffsetInSketcher"
+__author__  = "edi"
+__version__ = "0.03"
+__License__ = 'License identifier from https://spdx.org/licenses/, e.g. LGPL-2.0-or-later as FreeCAD, MIT, CC0-1.0'
+__Help__ = 'Select some lines in a sketch. Start the macro. Show offset value using the rubber band.'
+__Status__ = 'Stable'
+__Requires__ = 'e.g. FreeCAD >= v0.18'
+__date__    = "17/12/2020"
+
+from PySide import QtGui
+from pivy.coin import *
+if App.Version()[1].find('18') == 0: import DraftTrackers, Draft
```

Comment:
`if App.Version()[:2] == ['0', '19']: ...`

Reply:
I just tested versions 0.18 and 0.19. Don't know what happenes in earlier versions.